web/src/styles/main.scss

```diff
@@ -47,4 +47,14 @@ h2{
 
 .v-expansion-panel-content__wrap {
   padding: 12px 12px 0px !important;
-}
\ No newline at end of file
+}
+
+form[name="contact-speaker"] {
```

Comment:
It might be better to use Vuetify's built-in styling for textarea. Read more [here](https://vuetifyjs.com/en/components/textareas/#rows)

Reply:
I agree, removed the additional styles